rascaline/src/calculators/radial_basis/mod.rs

```diff
@@ -21,6 +21,16 @@ pub enum RadialBasis {
         #[serde(default = "serde_default_spline_accuracy")]
         spline_accuracy: f64,
     },
+    Tabulated {
+        // Provide user-defined splines. 
+        // These consist of the positions of the spline points (as a serialized 1D array),
+        // values of the radial integrals at the spline point for each l and each n (as a 
+        // serialized flattened 3D array), and derivatives of the radial integrals at the spline 
+        // points for each l and each n (as another serialized flattened 3D array).
+        positions: String,
```

Comment:
I think this would be better as a `Vec<f64>`, and have values/derivative as `Vec<Array2<f64>>` instead of adding more parsing down below. This could require a custom `deserialize_with` (https://serde.rs/field-attrs.html#serialize_with) and friends.

Reply:
Alternatively, this could also be 

```rust
pub enum RadialBasis {
    Gto {
        // ...  
    },
    Tabulated {
        points: Vec<SplinePoint>
    },
}


SplinePoint {
    positions: f64,
    #[serde(with=serialize_arrays)]
    values: Array2<f64>,
    #[serde(with=serialize_arrays)]
    derivatives: Array2<f64>,
}

mod serialize_arrays {
    todo!()
}
```